packages/shared/src/lib/socialMedia.ts

```diff
@@ -57,7 +57,7 @@ export const getSocialReferrer = (): SocialIconType | null => {
   if (host.includes('reddit.')) {
     return SocialIconType.Reddit;
   }
-  if (host.includes('x.') || host.includes('twitter.')) {
+  if (host.includes('x.') || host.includes('twitter.') || host === 't.co') {
```

Comment:
Non-blocking: I think we can just store this in an array and do something link below or something along these lines:

```
const twitterHosts = ['x.', 'twitter.', 't.co'];
if (twitterHosts.some(host.includes) {
  ..
}
```